src/LSP3/getProfileMetadata.ts

```diff
@@ -50,13 +69,15 @@ export const getProfileMetadata = async (unviersalProfile: UniversalProfile) =>
 
     let profileData: LSP3ProfileMetadata;
 
-    await fetch(profileDataURL)
-        .then(async (result) => await result.json())
-        .then((result) => (profileData = result));
+    await axios({
```

Comment:
Oh you're converting from fetch to axios. Note that workers only have fetch.

Reply:
I had some issues where fetch was missing in node 16 but present in node 18. But after updating to node 18 I managed to build but had some issues with tests, maybe we can double check and see if axios is really needed here.